### PR TITLE
Fix mapping of webhook connectors that is stored in tfstate

### LINKED ISSUE #388: [Bug] elasticstack_kibana_action_connector webhook config not stored correctly in state
**Describe the bug**
Using the new elasticstack_kibana_action_connector object ends in endloop reconcile loops because the config parameter is not stored correctly and thus produces a diff everytime.

**To Reproduce**
```
resource "elasticstack_kibana_action_connector" "webhook" {
  name = "webhook"
  config = jsonencode({
    url     = "https://some-webhook"
    method  = "post"
    hasAuth = false
    headers = {
      Content-Type = "application/json"
    }
  })
  secrets           = jsonencode({})
  connector_type_id = ".webhook"
}
```

First apply is fine but another plan will show this

```
  # elasticstack_kibana_action_connector.webhook will be updated in-place
  ~ resource "elasticstack_kibana_action_connector" "webhook" {
      ~ config             = jsonencode(
          ~ {
              + hasAuth = false
              + headers = {
                  + Content-Type = "application/json"
                }
              + method  = "post"
                # (1 unchanged attribute hidden)
            }
        )
        id                 = "default/some-id"
        name               = "webhook"
        # (7 unchanged attributes hidden)
    }
```

Checking the state we see only this:

```
resource "elasticstack_kibana_action_connector" "webhook" {
    config             = jsonencode(
        {
            url     = "https://some-webhook"
        }
    )
    connector_id       = "some-id"
    connector_type_id  = ".webhook"
    id                 = "default/some-id"
    is_deprecated      = false
    is_missing_secrets = false
    is_preconfigured   = false
    name               = "webhook"
    secrets            = jsonencode({})
    space_id           = "default"
}
```

**Expected behavior**
The whole config is stored in the state and another plan without changes will not show any diff

**Versions (please complete the following information):**
 - Terraform Version v1.5.2
 - Provider version 0.6.2
 - Elasticsearch Version 8.8.2

**Additional context**

### PR DESCRIPTION
**Pull request by btrieger**
Connectors were only properly storing url in the config in the tfstate file as they were being mapped to Tines. 

Fix #388

### CODE REVIEW

File: internal/kibana/connector_test.go [hunk 1]
```diff
@@ -1309,6 +1309,8 @@ func TestAccResourceKibanaConnectorWebhook(t *testing.T) {
 	  name         = "%s"
 	  config = jsonencode({
 		url = "https://elastic.co"
+  		hasAuth = true
+    	method = "post"
```
Comment (dimuon) [inline internal/kibana/connector_test.go]:
```suggestion
  		hasAuth = true
  		method = "post"
```

File: internal/kibana/connector_test.go [hunk 2]
```diff
@@ -1327,6 +1329,8 @@ func TestAccResourceKibanaConnectorWebhook(t *testing.T) {
 	  name         = "Updated %s"
 	  config = jsonencode({
 		url = "https://elasticsearch.com"
+		hasAuth = true
+    	method = "post"
```
Comment (dimuon) [inline internal/kibana/connector_test.go]:
```suggestion
		hasAuth = true
		method = "post"
```

**Review by dimuon — DISMISSED** [review]
Thanks for contributing!
LGTM 👍 besides minor formatting issue.

**Comment by btrieger** [comment]
Pushed an update with the formatting. Looks like you need to approve again. Odd that it looked fine in my ide

**Pull request merged**